application/src/main/java/org/opentripplanner/osm/tagmapping/NorwayMapper.java

```diff
@@ -618,6 +618,7 @@ else if (speedLimit >= 11.1f) {
 
     props.setCarSpeed("highway=living_street", 1.94f); // 7 km/h
     props.setCarSpeed("highway=pedestrian", 1.94f); // 7 km/h
```

Comment:
Could you do a favour and put in FinlandMapper car speed limit of `4.5f` for `highway=pedestrian` as the default here is slightly too slow?